Tasks/KubernetesManifestV0/task.loc.json

```diff
@@ -75,7 +75,9 @@
         "deploy": "deploy",
         "patch": "patch",
         "scale": "scale",
-        "delete": "delete"
+        "delete": "delete",
+        "reject": "reject",
+        "promote": "promote"
```

Comment:
Shouldn't the names be `promote canary` and `reject canary`, the names aren't quite obvious for a person who hasn't chosen a deployment strategy

Reply:
@shashankbarsin to take a look at names.

Reply:
Let's keep it generic promote and reject. In future we'll add more strategies like blue/greeen